packages/flutter/lib/src/material/stepper.dart

```diff
@@ -362,11 +378,21 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
     return false;
   }
 
-  Widget _buildLine(bool visible) {
+  Color _connectorColor(bool isActive) {
```

Comment:
If `widget.connectorColor` resolves to null, then we should use the default colors:
```dart
    final ColorScheme colorScheme = Theme.of(context).colorScheme;
    final Set<MaterialState> states = <MaterialState>{
      if (isActive) MaterialState.selected else MaterialState.disabled,
    };
    return widget.connectorColor?.resolve(states) ?? isActive ? colorScheme.primary : Colors.grey.shade400;
```

A test should confirm this.

Reply:
```
if(widget.connectorColor == null) {
   return isActive ? colorSchema.primary : Colors.grey.shade400;
}
```
this code set colors to default if `connectorColor` is null

Reply:
That fails if connectorColor is non-null but connectorColor.resolve(states) returns null. Please see https://github.com/flutter/flutter/pull/122485/files#r1149856878